# HG changeset patch
# User Language Forge
# Date 1545411484 -3600
#      Fri Dec 21 17:58:04 2018 +0100
# Branch 7000072
# Node ID 0f34bba8cac6078377cdfc0c3e4431a796f84562
# Parent  9a22d579d8b05ffc0ca29c2e00bda5939d90051f
[LfMergeBridge: 3.0.1.254] Language Forge: 1 entry added, 1 entry deleted

diff -r 9a22d579d8b0 -r 0f34bba8cac6 Linguistics/Lexicon/Lexicon_02.lexdb
--- a/Linguistics/Lexicon/Lexicon_02.lexdb	Fri Dec 21 17:56:42 2018 +0100
+++ b/Linguistics/Lexicon/Lexicon_02.lexdb	Fri Dec 21 17:58:04 2018 +0100
@@ -1,2 +1,88 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Lexicon />
\ No newline at end of file
+<Lexicon>
+	<LexEntry
+		guid="515442e1-d369-484a-a039-f05dfb8748c0">
+		<DateCreated
+			val="2018-12-21 16:48:36.215" />
+		<DateModified
+			val="2018-12-21 16:57:38.8" />
+		<DoNotUseForParsing
+			val="False" />
+		<HomographNumber
+			val="0" />
+		<LexemeForm>
+			<MoStemAllomorph
+				guid="0b05dc15-4f63-4251-a8dd-2497dcf39f6a">
+				<Form>
+					<AUni
+						ws="fr">lf1modified</AUni>
+				</Form>
+				<IsAbstract
+					val="False" />
+				<MorphType>
+					<objsur
+						guid="d7f713e8-e8cf-11d3-9764-00c04f186933"
+						t="r" />
+				</MorphType>
+			</MoStemAllomorph>
+		</LexemeForm>
+		<MorphoSyntaxAnalyses>
+			<MoStemMsa
+				guid="04176cfd-4ffa-476a-8277-bc4f068a2f8f">
+				<PartOfSpeech>
+					<objsur
+						guid="a8e41fd3-e343-4c7c-aa05-01ea3dd5cfb5"
+						t="r" />
+				</PartOfSpeech>
+			</MoStemMsa>
+		</MorphoSyntaxAnalyses>
+		<Senses>
+			<ownseq
+				class="LexSense"
+				guid="713cdbac-47f6-4bf9-9a11-2290e415ef3d">
+				<Definition>
+					<AStr
+						ws="en">
+						<Run
+							ws="en">Word added by LF</Run>
+					</AStr>
+				</Definition>
+				<Examples>
+					<ownseq
+						class="LexExampleSentence"
+						guid="2201a920-8793-417c-b40e-9b36690fb22b">
+						<Example>
+							<AStr
+								ws="fr">
+								<Run
+									ws="fr">C'est un exemple</Run>
+							</AStr>
+						</Example>
+						<Translations>
+							<CmTranslation
+								guid="ad0d905b-a29d-4652-b9c1-a79558282d61">
+								<Translation>
+									<AStr
+										ws="en">
+										<Run
+											ws="en">This is an example</Run>
+									</AStr>
+								</Translation>
+								<Type>
+									<objsur
+										guid="d7f7164a-e8cf-11d3-9764-00c04f186933"
+										t="r" />
+								</Type>
+							</CmTranslation>
+						</Translations>
+					</ownseq>
+				</Examples>
+				<MorphoSyntaxAnalysis>
+					<objsur
+						guid="04176cfd-4ffa-476a-8277-bc4f068a2f8f"
+						t="r" />
+				</MorphoSyntaxAnalysis>
+			</ownseq>
+		</Senses>
+	</LexEntry>
+</Lexicon>
\ No newline at end of file
diff -r 9a22d579d8b0 -r 0f34bba8cac6 Linguistics/Lexicon/Lexicon_09.lexdb
--- a/Linguistics/Lexicon/Lexicon_09.lexdb	Fri Dec 21 17:56:42 2018 +0100
+++ b/Linguistics/Lexicon/Lexicon_09.lexdb	Fri Dec 21 17:58:04 2018 +0100
@@ -1,55 +1,2 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Lexicon>
-	<LexEntry
-		guid="863d94c8-83ca-4748-8c9b-4730d704e10e">
-		<DateCreated
-			val="2018-12-21 16:46:16.647" />
-		<DateModified
-			val="2018-12-21 16:55:46.755" />
-		<DoNotUseForParsing
-			val="False" />
-		<HomographNumber
-			val="0" />
-		<LexemeForm>
-			<MoStemAllomorph
-				guid="590f2d93-27e5-4a0b-84d5-14e720c69e11">
-				<Form>
-					<AUni
-						ws="fr">flexmodified</AUni>
-				</Form>
-				<IsAbstract
-					val="False" />
-				<MorphType>
-					<objsur
-						guid="d7f713e8-e8cf-11d3-9764-00c04f186933"
-						t="r" />
-				</MorphType>
-			</MoStemAllomorph>
-		</LexemeForm>
-		<MorphoSyntaxAnalyses>
-			<MoStemMsa
-				guid="980fe1d4-aaa9-4c67-8716-de4097e53e71">
-				<PartOfSpeech>
-					<objsur
-						guid="46e4fe08-ffa0-4c8b-bf98-2c56f38904d9"
-						t="r" />
-				</PartOfSpeech>
-			</MoStemMsa>
-		</MorphoSyntaxAnalyses>
-		<Senses>
-			<ownseq
-				class="LexSense"
-				guid="d349956d-7004-46f8-a82e-865e96c9ac6e">
-				<Gloss>
-					<AUni
-						ws="en">created in FLEx</AUni>
-				</Gloss>
-				<MorphoSyntaxAnalysis>
-					<objsur
-						guid="980fe1d4-aaa9-4c67-8716-de4097e53e71"
-						t="r" />
-				</MorphoSyntaxAnalysis>
-			</ownseq>
-		</Senses>
-	</LexEntry>
-</Lexicon>
\ No newline at end of file
+<Lexicon />
\ No newline at end of file
